### ISSUE TITLE
MIOpenDriver Backwards Convolution Weights Failing

### ISSUE THREAD

**Issue by patflick** [ROCmSoftwarePlatform/MIOpen#11]
The `MIOpenDriver fails for the Backwards weights for all parameters I tried, including the default ones (see below). Is this reproducible on your side, or is there something wrong with my setup/compilation?

```
$ ./MIOpenDriver conv                          
MIOpenDriver: conv
Forward Convolution Verifies on CPU and GPU
Backward Convolution Data Verifies on CPU and GPU
Backward Convolution Weights Failed: 0.993599
```

**Comment by dagamayank** [comment]
@patflick I am pretty sure this will also work with -t 1 flag. At least does for me. Please confirm.

**Comment by dagamayank** [comment]
Fixed in 6c254cd42e514132ddc398072781389848bc0cdf
